scripts/custom/clean_osiris_codes.py

```diff
@@ -0,0 +1,20 @@
+from scripts import utils
+import re
+
+
+# function clean_osiris_codes
+# remove all chars before and including the first colon
+
+# OSIRIS database has other codes that are not accepted by FHIR : ["OSIRIS:",]
+
+osirisCodes = ["HL7:", "UMLS:", "LOINC:", "ATC:"]
+
+
+def clean_osiris_codes(raw_input):
+    if utils.is_empty(raw_input):
+        return None
+
+    code = re.search("[A-z0-9]*:", raw_input).group(0)
```

Comment:
This will return an error if the raw_input is not of type "TERMINOLOGY:code". 
Maybe you could do:
`code = re.match(r"[A-z0-9]*:(.*)", raw_input)`
`if code in osirisCodes: return code.group(1)`